R/pkg/R/generics.R

```diff
@@ -1054,6 +1054,10 @@ setGeneric("year", function(x) { standardGeneric("year") })
 #' @export
 setGeneric("glm")
 
+#' @rdname predict
+#' @export
+setGeneric("predict", function(object, newData) { standardGeneric("predict") })
```

Comment:
It may be better to keep the signature of "predict" to be same as that in R base.
```
setGeneric("predict", function(object, ...) { standardGeneric("predict") })
```